index.js

```diff
@@ -10,8 +10,11 @@ module.exports = function (grunt, opts) {
 
 	var pattern = arrify(opts.pattern || ['grunt-*', '@*/grunt-*']);
 	var config = opts.config || pkgUp.sync();
+
 	var scope = arrify(opts.scope || ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies']);
 
+	var dirName = (typeof config === 'string') ? path.dirname(path.resolve(config)) : __dirname;
```

Comment:
I think you meant `process.cwd()` there, not `__dirname`. With `__dirname` it would resolve from this package.

Also no need for the parens.